shopfloor_cluster_picking_repack/__manifest__.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2021 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Shopfloor Packing",
+    "version": "18.0.1.0.0",
+    "summary": """ Manage Packing into cluster picking""",
+    "author": "ACSONE SA/NV,Odoo Community Association (OCA)",
```

Comment:
I should have been listed here also 
```suggestion
    "author": "ACSONE SA/NV,BCIM,Odoo Community Association (OCA)",
```